pkg/install/v1alpha1/cstor_pool.go

```diff
@@ -268,6 +268,9 @@ spec:
               postStart:
                  exec:
                     command: ["/bin/sh", "-c", "sleep 2"]
+              preStop:
+                 exec:
+                    command: ["/bin/bash", "-c", "rm /tmp/uzfs.sock"]
```

Comment:
Can we set `/tmp/uzfs.sock` as castemplate defaultConfig?
It would be better to have this info coming from ENV variable & used here as well as inside the logic.
If remove errors then do we get any logs?
Will this command work in various environments with strict security policies?

Reply:
Since `zrepl` process inside the cstor-pool container creates a sock file and mount on the host path(/var/openebs/shared-<pool_name>) as a cleanup process before cstor-pool container removes sock file, we are removing sock file as a preStop hook inside the cstor-pool container (which will affect at host path also) it means it will work in all environments but I am not sure if environment have strict security policies.